docs/en/operations/table_engines/custom_partitioning_key.md

```diff
@@ -1,46 +1,36 @@
-
 # Custom Partitioning Key
 
-The partition key can be an expression from the table columns, or a tuple of such expressions (similar to the primary key). The partition key can be omitted. When creating a table, specify the partition key in the ENGINE description with the new syntax:
+Partitioning is available for the [MergeTree](mergetree.md) family tables (including [replicated](replication.md) tables). [Materialized views](materializedview.md) based on MergeTree table support partitioning as well.
 
-```
-ENGINE [=] Name(...) [PARTITION BY expr] [ORDER BY expr] [SAMPLE BY expr] [SETTINGS name=value, ...]
-```
+A partition is a logic combination of records in a table by a specified criterion. You can set a partition by an arbitrary criterion. For example: by month, by day, by event type and so on. Each partition is stored separately in order to simplify manipulations with this data. When accessing the data ClickHouse only uses the corresponding partition.
 
-For MergeTree tables, the partition expression is specified after `PARTITION BY`, the primary key after `ORDER BY`, the sampling key after `SAMPLE BY`, and `SETTINGS` can specify `index_granularity` (optional; the default value is 8192), as well as other settings from [MergeTreeSettings.h](https://github.com/yandex/ClickHouse/blob/master/dbms/src/Storages/MergeTree/MergeTreeSettings.h). The other engine parameters are specified in parentheses after the engine name, as previously. Example:
+The partition is specified in the clause `PARTITION BY expr` when [creating a table](mergetree.md#table_engine-mergetree-creating-a-table). The partition key can be any expression from the table columns. To specify the traditional partitioning by month, use an expression `toYYYYMM(date_column)`. For example:
```

Comment:
* clause `PARTITION BY expr` -> `PARTITION BY expr` clause
* it's not obvious why by month is "traditional", I'd try to avoid this word